pom.xml

```diff
@@ -59,10 +59,15 @@
 	<properties>
 		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
 		<project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
-		 <dockerfile-maven.version>1.3.6</dockerfile-maven.version>
-		  <project.docker.repository>scalecube/configuration</project.docker.repository>
+		<dockerfile-maven.version>1.3.6</dockerfile-maven.version>
+		<project.docker.repository>scalecube/configuration</project.docker.repository>
 		<argLine>-Xms1024m -Xmx2048m</argLine>
 
+		<scalecube-services.version>2.0.26</scalecube-services.version>
+		<reactor-core.version>3.1.6.RELEASE</reactor-core.version>
+		<redisson.version>3.4.3</redisson.version>
+		<jackson-databind.version>2.7.6</jackson-databind.version>
```

Comment:
Try to use the jackson version which one used in scalecube-services (I guess it's 2.9.5).